pkg/capabilities/capabilities.go

```diff
@@ -4,47 +4,73 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/syndtr/gocapability/capability"
+	"kernel.org/pub/linux/libs/security/libcap/cap"
 )
 
 var (
-	// NewPid2 is not defined on an interface
+	// GetPID is not defined on an interface
 	// therefore it must be patched in for testability
-	NewPid2 = capability.NewPid2
+	GetPID = func(pid int) (ICapabilitiesSet, error) {
+		return cap.GetPID(pid)
+	}
```

Comment:
Can you describe to me the reason you've designed it this way? What's the purpose of the interface?

Reply:
The purpose here is to be able to test the functions in this file.
Because the `cap` library doesn't export any interface, to be able to mock its functionality I needed to use a variable that points to the function. This method is used broadly in the project, you can see it in other files.
About the interface, I just need a way to mock the capabilities used. Again, it's all for testability reasons.